lib/screens_web/views/v2/audio/reconstructed_alert_view.ex

```diff
@@ -109,4 +109,9 @@ defmodule ScreensWeb.V2.Audio.ReconstructedAlertView do
       _ -> ""
     end
   end
+
+  defp adjust_remedy(remedy) when remedy === "Seek alternate route",
+    do: "Please seek an alternate route"
```

Comment:
Nothing wrong with this, but can be done without a guard.

```suggestion
  defp adjust_remedy("Seek alternate route"), do: "Please seek an alternate route"
```